### ISSUE TITLE
Gravity and Simulation Speed (Timescale) keying has no effect on simulation.

### ISSUE THREAD

**Issue by fineskimo** [sebbas/blender-mantaflow#12]
Hi there, great work so far.

Compiled a build on 07/12/17 and ran a few simulations, I noticed that keyframing the value of the timescale/simulation speed or gravity in the Domain settings had no effect during the simulation.

Also Scene Gravity appears not to effect the simulation at all.

Hope this is helpful.

Fin

**Comment by sebbas** [comment]
Hi, sorry for the late reply. But better late than never :)

I see what you mean! And it's, in fact, a really good idea! Besides the gravity, there are even a couple more variables that could be parsed "dynamically" and this way made available for keyframing.

So thanks for the suggestion! I'll get back once it's implemented!

**Comment by fineskimo** [comment]
Thanks for getting back to me on this... looking forward to see which
variables you open up for dynamic parsing...

**Comment by GottfriedHofmann** [comment]
Any updates on this? Keyframing the time scale would be really appreciated! In fact please try to parse as many parameters dynamically as possible since that is also one of the core principles of Blender: Everything can be animated :)

**Comment by sebbas** [comment]
Yes! Finally integrated this into the (new) baking setup that I am working on right now. 

All variables that Mantaflow uses are now updated at every frame before the simulation "step". (2a6a579b06970ee8c37e2485cccfe016f19151e8, 7d4c961ddefe03257de7cc70d72da1f5095b8bee). I hope this covers all properties from the UI - I just tested it with gravity and timescale for now.

Invert gravity for smoke plume:
![manta_keyfraed_gravity](https://user-images.githubusercontent.com/1890600/37874669-d24d23f6-3033-11e8-96a8-cf9e505fe1c5.gif)

Slow-motion for liquid (FLIP only, no mesh) when it hits bottom of domain:
![manta_keyframed_timescale](https://user-images.githubusercontent.com/1890600/37874984-14d3e7c8-3039-11e8-866a-01a5cadffdd6.gif)

**Comment by fineskimo** [comment]
Excellent... will run some tests during the week...

**Comment by no-tably** [comment]
Any more news on this thread?
